app/views/viewTransfers.scala.html

```diff
@@ -65,6 +65,38 @@ <h1 class="govuk-heading-l">View Transfers</h1>
         }
       </table>
 
+      @if(consignments.nonEmpty && totalPage.exists(_ > 1)) {
+        <nav class="govuk-pagination govuk-pagination__center" role="navigation" aria-label="results">
+          <div class="govuk-pagination__prev" @{if(pageNumber < 2) "hidden"}>
+            <a class="govuk-link govuk-pagination__link" href="@routes.ViewTransfersController.viewConsignments(pageNumber - 1)" rel="prev">
+              <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+                <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+              </svg>
+              <span class="govuk-pagination__link-title">Previous</span></a>
+          </div>
+          <ul class="govuk-pagination__list">
+          @for(page <- 1 to totalPage.getOrElse(1)) {
+            @if(pageNumber == page) {
+              <li class="govuk-pagination__item govuk-pagination__item--current">
+                <a class="govuk-link govuk-pagination__link" href="#" aria-label="Page @page" aria-current="page">@page</a>
+              </li>
+            } else {
+              <li class="govuk-pagination__item">
+                <a class="govuk-link govuk-pagination__link" href="@routes.ViewTransfersController.viewConsignments(page)" aria-label="Page @page">@page</a>
+              </li>
+            }
+          }
+          </ul>
+          <div class="govuk-pagination__next" @{if(totalPage.exists(_ <= pageNumber)) "hidden"}>
+            <a class="govuk-link govuk-pagination__link" href="@routes.ViewTransfersController.viewConsignments(pageNumber + 1)" rel="next">
+              <span class="govuk-pagination__link-title">Next</span>
+              <svg class="govuk-pagination__icon govuk-pagination__icon--next" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+                <path d="m8.107-0.0078125-1.4136 1.414 4.2926 4.293h-12.986v2h12.896l-4.1855 3.9766 1.377 1.4492 6.7441-6.4062-6.7246-6.7266z"></path>
+              </svg>
+            </a>
+          </div>
+        </nav>
+      }
```

Comment:
For users with many consignments the pagination looks like so: 
![Screenshot from 2023-03-29 10-58-09](https://user-images.githubusercontent.com/9700622/228500238-390d5d6b-9ecb-4c1e-b47e-2f4cad0a5229.png)

Probably need to have some logic for handling large pagination refer to the **For larger numbers of pages** section https://design-system.service.gov.uk/components/pagination/.

Reply:
I just remembered something about the large pagination. Say if you have 100 pages and you were on page 50. So with GDS implementation it would look like `<- Previous 1 ... 45 46 47 48 49 **50** 51 52 53 54 55 ... 100 Next ->`
Now the user wants to go to page 20. There's no efficient way of doing that without spamming the Previous button. Probably a question for Gareth.

Reply:
Good point, I have added the ellipses for the larger number of pages and Gareth is happy too about this change.